aspnetcore/publishing/iis.md

```diff
@@ -170,41 +174,64 @@ In web farm scenarios, an application can be configured to use a UNC path to sto
+One quick way to determine if the application is working properly is to run the application directly on Kestrel. If the application was published as a portable app, execute **dotnet my_application.dll** in the deployment folder. If the application was published as a self-contained app, run the application's executable directly from a command prompt, **my_application.exe**, in the deployment folder. If Kestrel is listening on default port 5000, you should be able to browse the application at `http://localhost:5000/`. If the application responds normally at the Kestrel endpoint address, the problem is more likely related to the IIS-ASP.NET Core Module-Kestrel configuration and less likely within the application itself.
 
-A way to determine if the IIS reverse proxy to the Kestrel server is working properly is to perform a simple static file request for a stylesheet, script, or image from the application's static assets in *wwwroot* using [Static File middleware](../fundamentals/static-files.md). If the application can serve static files but MVC Views and other endpoints are failing, the problem is less likely related to the IIS-ASP.NET Core Module-Kestrel configuration and more likely within the application itself (for example, MVC routing or 500 Internal Server Error).
+One way to determine if the IIS reverse proxy to the Kestrel server is working properly is to perform a simple static file request for a stylesheet, script, or image from the application's static files in *wwwroot* using [Static File middleware](../fundamentals/static-files.md). If the application can serve static files but MVC Views and other endpoints are failing, the problem is less likely related to the IIS-ASP.NET Core Module-Kestrel configuration and more likely within the application itself (for example, MVC routing or 500 Internal Server Error).
+
+When Kestrel starts normally behind IIS but the app won't run on the server after successfully running locally, you can temporarily add an env var to *web.config* to set the `ASPNETCORE_ENVIRONMENT` to `Development`. As long as you don't override the environment in app startup, this will allow the [developer exception page](../fundamentals/error-handling.md) to appear when the app is run on the server. Setting the environment env var in this way is only recommended for staging/testing servers that are not exposed to the Internet. Be sure you remove the env var from the *web.config* file when finished. For information on setting env vars via *web.config* for the reverse proxy, see [environmentVariables child element of aspNetCore](../hosting/aspnet-core-module.md#environmentvariables-child-element-of-aspnetcore).
```

Comment:
> note extra env in one instance "environment env var"

Side note: That was done on purpose, because they would be setting the "Environment" env var. However, you're right ... that choice of words blows. I'm fixing it now.